addon/components/base/bs-button.js

```diff
@@ -467,7 +458,9 @@ export default class Button extends Component {
       }
     }
 
-    return this.get('bubble');
+    if (!this.get('bubble')) {
+      e.stopPropagation();
+    }
```

Comment:
Oh, I wasn't aware that return value is not controlling event propagation anymore if used with `{{on}}` listener. Good catch! :clap:

Reply:
Yeah, that's a feature of the older Ember events / EventDispatcher, not a thing when using native `addEventListener()` that `{{on}}` is using.

Also a thing that the codemod is not doing correctly in this case. Wonder if we can/should add that code automatically!? 🤔

Reply:
> Also a thing that the codemod is not doing correctly in this case. Wonder if we can/should add that code automatically!? thinking

Not sure if automation is worse the time. But it should be at least mentioned in the docs. I think the codemod needs a section in the docs which lists all the possible edge cases that may be broken after conversion. The event bubbling is only of them, isn't it?